include/robots/gazebo/omni2d.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+// Third-party includes
+#include "third_party/units.h"
+
+// Gazebo
+#include <gazebo/msgs/msgs.hh>
+#include <gazebo/transport/transport.hh>
+
+namespace BoBRobotics {
+namespace Robots {
+namespace Gazebo {
+using namespace units::literals;
+
+class Omni2D
```

Comment:
Might it be nice to add a base class for omniwheels as (in various PRs) we now have 2?